dotCMS/src/main/java/com/dotcms/graphql/InterfaceType.java

```diff
@@ -23,12 +23,25 @@
 import static graphql.Scalars.GraphQLString;
 
 import com.dotcms.contenttype.model.type.BaseContentType;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.contenttype.model.type.EnterpriseType;
+import com.dotcms.contenttype.model.type.FileAssetContentType;
+import com.dotcms.contenttype.model.type.FormContentType;
+import com.dotcms.contenttype.model.type.KeyValueContentType;
+import com.dotcms.contenttype.model.type.PageContentType;
+import com.dotcms.contenttype.model.type.PersonaContentType;
+import com.dotcms.contenttype.model.type.SimpleContentType;
+import com.dotcms.contenttype.model.type.VanityUrlContentType;
+import com.dotcms.contenttype.model.type.WidgetContentType;
+import com.dotcms.enterprise.LicenseUtil;
+import com.dotcms.enterprise.license.LicenseLevel;
 import com.dotcms.graphql.datafetcher.FolderFieldDataFetcher;
 import com.dotcms.graphql.datafetcher.LanguageDataFetcher;
 import com.dotcms.graphql.datafetcher.SiteFieldDataFetcher;
 import com.dotcms.graphql.datafetcher.TitleImageFieldDataFetcher;
 import com.dotcms.graphql.datafetcher.UserDataFetcher;
 import com.dotcms.graphql.resolver.ContentResolver;
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'com.dotcms.repackage.com.google.common.annotations.VisibleForTesting'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4740432)

Reply:
Also, let's stay away from repackaged classes when possible
If I had to mark my method as `VisibleForTesting` I'd rather using  **com.google.common.annotations.VisibleForTesting;**

Hopefully Repackaged version someday will go away for good